### DISCUSSION TITLE
No parameter named "bootstyle"

### DISCUSSION THREAD

**Discussion by Swarkin** [israel-dryer/ttkbootstrap#391] [Q&A] [answered]
The bootstyle isnt a valid parameter for any widget constructor.
I just downloaded ttkbootstrap and copied this example from the docs. It doesn't seem to work...

![image](https://user-images.githubusercontent.com/102416174/208749457-a4fd20d5-7429-44d9-8c4e-2b28dddf19d6.png)

**Comment by rdbende** [comment]
What version of ttkbootstrap are you using?

**Reply by Swarkin** [reply]
1.10.0. I installed it using pip install ttkbootstrap, so this should be the latest.

**Reply by rdbende** [reply]
Then it's really strange, I have the same version, and I doesn't have any problem with your code above.

**Reply by Swarkin** [reply]
I found out that insted of bootstyle it's just called style. Changing the parameter to this seems to work without problems.

**Reply by rdbende** [reply]
It works for me with `bootstyle` though.

**Reply by aDramaQueen** [reply, marked as answer]
I have this behavior too. I did it in 2 variants:
- Button 1 with bootstyle
- Button 2 with style

Both of them work though, but it's seems to be a problem with the type hinting.

![grafik](https://user-images.githubusercontent.com/52710085/208785875-ac82c874-376d-4adb-9b3b-83100020a78b.png)
